sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -36,4 +36,12 @@
     /** Error while session is closed.*/
     public static final ServiceBusErrorSource CLOSE_SESSION = fromString("CLOSE_SESSION",
         ServiceBusErrorSource.class);
+
+    /** Error while sending a message.*/
+    public static final ServiceBusErrorSource SENDING = fromString("SENDING",
```

Comment:
To be consistent with the usage of other terms here, we should just call this `SEND`. Also, we should do the same for `ABANDONED` -> `ABANDON` (sorry, this was not part of your PR).

Reply:
I live to serve. No apologies needed ;)